### ISSUE TITLE
Console crashes when booting with Grandia (SCUS-94457)

### ISSUE THREAD

**Issue by raynebc** [pathartl/BleemSync#136]
As mentioned in issue #126, if I start with a blank flash drive, extract BleemSync 0.4.1 onto it, add this one game and run BleemSync.exe, it adds the game to the database.  However when I boot the PS classic with this flash drive in it, the console powers itself off during the initial Sony logo screen.  Powering on the console again boots up the games that are built-into the console because the flash drive's file system is corrupt due to the crash.

I can set up this same flash drive with 19 other games that I have so the hardware and the steps used are working.  I obtained the .bin file from multiple sources (each decompressed with zero errors so there was no corruption during download).  I can run the affected Grandia bin and cue files in ePSXe so those files are working.  There seems to be some kind of limitation with BleemSync and/or the PS classic and this game.

**Comment by pathartl** [comment]
Sounds like a damaged USB device, not using a powered hub, or didn't disable current limiting. Not in the scope of BleemSync.

**Comment by ExSOLDIER1** [comment]
There is no limitation with either. I am running it perfectly on mine using Bleemsync 0.4.1. What kind of Flash drive are you using and how old is it? There is a chance that if it is old certain sectors can be corrupt which is why it is working on PC but not off your flash drive.

**Comment by raynebc** [comment]
The USB device (a brand new Samsung 64GB FIT Plus USB 3.1 flash drive) isn't expected to be damaged in any way.  This crash only happens when that specific game is injected to the database, regardless of how many others are present on the flash drive.  I launched several games from this flash drive after removing Grandia and they all more or less emulate flawlessly.  This generally disproves it being an issue with insufficient power to the flash drive or a damaged USB device.

I'll run a full chkdisk to prove there are no bad sectors, but if the results show there are none, I'll have to open a new ticket because this project won't let me re-open a prematurely closed ticket.

**Comment by raynebc** [comment]
chkdsk and fc confirm the flash drive and the game image were fine.  After testing with a powered USB hub, it does seem to work fine even when I put all 20 of the games (including Grandia) back on.  So while the flash drive is perfectly fine, the limitation is that the console provides so little power to each of its USB ports that it might not be enough to handle a flash drive.  It's not at all clear why this was the only of 20 games that I tried that caused the console to crash in this way.  It's not even the largest .bin file among those that I've tested.
